src/vault.ts

```diff
@@ -82,8 +87,8 @@ export async function openVault(progress: Progress<{ message: string }>, vaultFi
   progress.report({ message: 'Searching for Vault configuration...' })
   try {
     const ansibleCfgFile = await findAnsibleConfigurationFile(vaultFile)
-    const passwordFile = await parseVaultPasswordFilePath(ansibleCfgFile)
-    progress.report({ message: 'Found Vault configuration, decrypting...' })
+    const passwordFile = await findVaultPasswordFilePath(ansibleCfgFile)
```

Comment:
I tested `ansible-vault` from the command line and it seems if you have `ANSIBLE_VAULT_PASSWORD_FILE` set in the environment, it overrides any values in `ansible.cfg`. So in that case we don't even have to search for the configuration file, so we could skip calling `findAnsibleConfigurationFile` altogether.

I would maybe extract a separate function for resolving the password file, something like this:

```typescript
async function resolveVaultPasswordFile() {
  if (process.env.ANSIBLE_VAULT_PASSWORD_FILE) {
    progress.report({ message: 'Using Vault password file from the environment (ANSIBLE_VAULT_PASSWORD_FILE)' })
    return Uri.parse(process.env.ANSIBLE_VAULT_PASSWORD_FILE)
  } else {
    const ansibleCfgFile = await findAnsibleConfigurationFile(vaultFile)
    return parseVaultPasswordFilePath(ansibleCfgFile)
  }
}
```

and within `openVault` we'd simply have something like

```typescript
   try {
     const await passwordFile = resolveVaultPasswordFile()
     return await decryptVault(ansibleCfgFile, passwordFile, vaultFile)
   } catch() { ... } 
```